solid/templates/runscript.sh

```diff
@@ -87,7 +87,7 @@ echo -e "\n"
 
 rm ${inputfile}
 rm mysql-connector-c++-1.1.8-linux-glibc2.5-x86-64bit.tar.gz
-rm S2-histos_cycleMode_${jobnumber}_$runNUMBER.root S2-tuple_${jobnumber}_$runNUMBER.root log_${jobnumber}_$runNUMBER.txt
```

Comment:
Do you not need the jobnumber and runnumber any more?

Reply:
yes, we need job and runnumber for saving them on the SE, for removing them we don't need them.